cmd/triage/notify.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	jira "github.com/andygrunwald/go-jira"
+)
+
+func notify(slackHook string, slackClient *http.Client, issue jira.Issue, assignee TeamMember) error {
+	var msg bytes.Buffer
+	err := json.NewEncoder(&msg).Encode(struct {
+		LinkNames bool   `json:"link_names"`
+		Text      string `json:"text"`
+	}{
+		LinkNames: true,
+		Text:      "<@" + assignee.SlackId + "> you have been assigned the triage of this bug: " + jiraBaseURL + "browse/" + issue.Key,
```

Comment:
Do you think we should differentiate from the pre-triage message to highlight this is a reminder?

Reply:
I thought about that, but couldn't come up with a better message. Any suggestions?

Reply:
What do you think of "@slack_user gentle reminder to triage these bugs: list of bugs"?

Reply:
Done